pkg/driver/builder.go

```diff
@@ -73,6 +82,9 @@ func (b *Builder) GetDriver(version int, filePath string) (core.Driver, error) {
 				logger:           b.Logger,
 				ExecutionManager: b.ExecutionManager,
 			},
+			goInstaller: GoInstaller{
```

Comment:
Instead of passing of passing `nodeInstaller` and `goInstaller` , can we create a composite struct  which has both say `languageVersionInstaller`
We can have a function install which takes struct like ```{ NodeVersion : <> , GoVersion : <>} ``` and install all required language versions
@saurabh-prakash what's your view on this ?